src/get_v2_vs_frac.py

```diff
@@ -67,27 +67,44 @@ def v2_vs_frac(Dmeson, ptMins, ptMaxs, CutSets, rawYieldFiles, fracFiles, output
     for iPt, (ptMin, ptMax) in enumerate(zip(ptMins, ptMaxs)):
         ptCent = (ptMin + ptMax) / 2
         nSets = CutSets[iPt]
+        
+        ptBinFrac = iPt + 1
```

Comment:
Hi @Marcellocosti! Thanks for development. Here you mean that we will have n configurations for the ry and just one for the fraction, right?